libretroshare/src/services/p3gxscircles.cc

```diff
@@ -278,8 +278,9 @@ bool p3GxsCircles::getCirclesSummaries(std::list<RsGroupMetaData>& circles)
 	uint32_t token;
 	RsTokReqOptions opts;
 	opts.mReqType = GXS_REQUEST_TYPE_GROUP_META;
-	if( !requestGroupInfo(token, opts)
-	        || waitToken(token) != RsTokenService::COMPLETE ) return false;
+	if( !requestGroupInfo(token, opts) || waitToken(token) != RsTokenService::COMPLETE )
+        cancelRequest(token);
```

Comment:
AFAIU due to the change in rsgxsifacehelper.h:304 this is not needed anymore

Reply:
I fixed it.